cosmos-init/plugins/packages/cosmosc2-tool-limitsmonitor/src/tools/LimitsMonitor/LimitsControl.vue

```diff
@@ -316,6 +316,12 @@ export default {
     },
     handleMessages(messages) {
       for (let message of messages) {
+        // We only want to handle LIMITS_CHANGE messages
+        // NOTE: The channel also sends LIMITS_SETTINGS and LIMITS_SET messages
+        if (message.type != 'LIMITS_CHANGE') {
+          continue
+        }
```

Comment:
This ultimately fixed the bug in LimitMonitor. It was processing the other types of messages and just throwing 'undefined' on the list.